examples/jsm/controls/OrbitControls.js

```diff
@@ -248,6 +250,7 @@ class OrbitControls extends EventDispatcher {
 				offset.applyQuaternion( quatInverse );
 
 				position.copy( scope.target ).add( offset );
+				scope.object.parent ? scope.object.parent.worldToLocal( position ) : scope.object.position.copy( position )
```

Comment:
`scope.object.parent.worldToLocal( position )` does not update `scope.object.position`. Can you try this please:
```js
if ( scope.object.parent !== null ) {

    scope.object.worldToLocal( position );

}

scope.object.position.copy( position );
```